Zend/zend_API.h

```diff
@@ -37,7 +37,7 @@ typedef struct _zend_function_entry {
 	zif_handler handler;
 	const struct _zend_internal_arg_info *arg_info;
 	uint32_t num_args;
-	uint32_t flags;
+	zend_fn_flags flags;
```

Comment:
I think it would be easier to introduce yet another filed (e.g. flags2).

Reply:
It would be easier indeed, but also a bit more confusing (i.e. having to remember which field is stored in which flag). This can be partially mitigated by naming the flags accordingly (e.g. `ZEND_ACC2_*`). But ok, Niels has suggested the same thing, so let me implement that instead.